src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+document.addEventListener('click', function (e) {
```

Comment:
Consider adding a check to ensure that the event target is indeed an anchor element before proceeding. This will make the code more robust.